package.json

```diff
@@ -106,6 +57,7 @@
     "sinon": "^1.15.4",
     "sinon-chai": "^2.8.0",
     "static-site-generator-webpack-plugin": "^1.2.0",
+    "webpack": "1.10.0",
```

Comment:
Any reason this (and `victory-util`) are pinned to exact versions? When it was up in prod `dependencies` this was `"^1.10.0"`. @ryan-roemer recommends potentially pinning the `builder` stuff for now, but everything else should be `^` unless you've got a reason.